### PR TITLE
Etcd hostname variable

### PR DESCRIPTION
**Pull request by sw0x2A**
This PR makes the etcd instance name configurable.

It was always set to be the hostname before. We need it to be the FQDN instead.

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -360,6 +365,7 @@
   Boolean $manage_etcd                         = true,
   Optional[String] $kube_api_advertise_address = undef,
   Optional[String] $etcd_version               = '3.1.12',
+  Optional[String] $etcd_hostname              = $hostname,
```
Comment (carabasdaniel) [inline manifests/init.pp]: Hi @sw0x2A,

Could you please change the hostname to use the new syntax for the facts, similar to line 350 ? 

Thanks.

Comment (sw0x2A) [reply manifests/init.pp]: @carabasdaniel Done.

Comment (carabasdaniel) [reply manifests/init.pp]: If you want the etcd hostname to be FQDN the variable should be $facts['fqdn']

Comment (sw0x2A) [reply manifests/init.pp]: I want to be able to set the etcd instance name to $fqdn and therefore I introduced a parameter for it in this PR. My intention is not to change the default. Instance name is currently set to $hostname in the template of etcd config file which is fine for most users of this module and changing the default might even be a breaking change.

**Comment by carabasdaniel** [comment]
👍

**Pull request merged**